### PR TITLE
help: Warp1: update

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
reviewing language

### CODE REVIEW

File: HelpSource/Classes/Warp1.schelp [hunk 1]
```diff
@@ -3,7 +3,7 @@ summary:: Warp a buffer with a time pointer
 categories:: UGens>Buffer, UGens>Generators>Granular
 
 description::
-Inspired by Chad Kirby's SuperCollider2 Warp1 class, which was inspired by Richard Karpen's sndwarp for CSound. A granular time stretcher and pitchshifter.
```
Comment (snappizz) [inline HelpSource/Classes/Warp1.schelp]: What happened to Chad Kirby?

**Comment by tiagmoraismorgado** [comment]
removing it, was the only way i got gitbook to tell me the sentence was hard to read. i think it sounds more clear like this, because you are repeating that the whole thing was referenced something twice, and you are repeating expressions, but if you think there is a better way to do so, tell me. :)

File: HelpSource/Classes/Warp1.schelp [hunk 2]
```diff
@@ -3,7 +3,7 @@ summary:: Warp a buffer with a time pointer
...
+UGen inspired by Chad Kirby's SuperCollider2 Warp1 class, itself referenced on Richard Karpen's sndwarp (CSound). Consists of a granular time stretcher with pitchshifting features.
```
Comment (vivid-synth) [inline HelpSource/Classes/Warp1.schelp]: I find this wording less clear than the original.

**Comment by vivid-synth** [comment]
Gitbook has an automated "readability-estimator"? Can you link to it? Thanks! @tiagmoraismorgado

File: HelpSource/Classes/Warp1.schelp (hunk 2 above)
Comment (tiagmoraismorgado) [inline HelpSource/Classes/Warp1.schelp]: so we close the thread

**Pull request closed without merging**